src/ontology/mondo-edit.obo

```diff
@@ -20523,6 +20523,8 @@ xref: SCTID:397732007 {source="DOID:12143"}
 xref: SCTID:398064005 {source="MONDO:equivalentTo", source="DOID:12143"}
 xref: UMLS:C0005697 {source="MONDO:equivalentTo", source="DOID:12143"}
 is_a: MONDO:0006026 {source="DOID:12143", source="MESH:D001750"} ! urinary bladder disease
+intersection_of: MONDO:0006026 ! urinary bladder disease
+intersection_of: disease_has_feature HP:0000011 ! Neurogenic bladder
```

Comment:
this suggests this may be a phenotype not a disease

Reply:
right, that question came up before. Should we make 'neurogenic disease' a candidate for obsoletion? I can create a ticket